ios/DatabaseHelper/DatabaseHelper/DatabaseHelper.swift

```diff
@@ -8,8 +8,6 @@
 import Foundation
 import SQLite3
 
-// TODO: This should be exposed to Objective-C in order to handle
```

Comment:
the entire folder for `DatabaseHelper` can be deleted as is not being used anywhere.